package.json

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Sorry to be a pedant, but can this extra line die while you're modifying things? ;)